app/views/jobseekers/passwords/edit.html.haml

```diff
@@ -9,6 +9,6 @@
 
       = f.govuk_error_summary
 
-      = f.govuk_password_field :password, label: { text: t(".new_password"), size: "s" }, width: "two-thirds"
+      = f.govuk_password_field :password, label: { text: t(".new_password"), size: "s" }, hint: { text: t("helpers.hint.jobseeker_sign_up.password") }, width: "two-thirds"
```

Comment:
You shouldn't define the hint here. It should come directly from the the translations